### ISSUE TITLE
Convert ParameterManager outputs to log4cplus

### ISSUE THREAD

**Issue by jbrown725** [UWB-Biocomputing/Graphitti#217]
ParameterManager currently outputs errors using cerr. These should be converted to log4cplus outputs for consistency with the rest of the console output.

**Comment by VivekGandhi54** [comment]
Should these all just be fatal? We never check for successful reads, so we always end up using uninitialized variables, which is bad.

**Comment by jbrown725** [comment]
Yeah true, if the parameter isn't in the config file then the simulation will probably be wrong and the simulator should exit rather than continue.

**Comment by jbrown725** [comment]
Related to #196. There are more files using cerr and cout than just the parameter manager.

**Comment by VivekGandhi54** [comment]
Side note:
Whoever does this should also probably clean up `FixedLayout::loadParameters()`, `DynamicLayout::loadParameters()`, and `Layout911::loadParameters()` because some of that code will become redundant.
